### DISCUSSION TITLE
contribution calendar, Helpp<title>

### DISCUSSION THREAD

**Discussion by Mucreth** [community/community#43325] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello there,
Although I submitted files to the repo, the boxes are not checked in my contribution calendar on the home page. can you help?
I did what was said in the links below but it didn't work :(

https://www.nikitakazakov.com/fix-githubs-missing-green-boxes-on-commits/
https://medium.com/@soufianerafik/commits-not-showing-up-on-github-contributions-graph-c3b539ad42a1

**Comment by samharp** [comment]
If you have already confirmed your email and repo settings are accurate, another thing to check is that you are either committing the code to the main/master branch, or that once your contributions are good to push, then you are merging to the main/master branch. The GitHub contribution graph doesn't show contributions made to forked repos or to other branches UNTIL they are merged to main/master. Once you merge the branch to main, then those commits will show on the contribution graph (on the day you made those commits, they won't all show on the day you merged the branch).

I hope this answers your question! 🙂
